styles/navigation/navbar/navbar.scss

```diff
@@ -0,0 +1,30 @@
+// Navbar
+//
+// A simple component to get navigation links to appear in one line. This is
+// mainly used to create navigation for small subsections on the application.
+//
+// Markup: navbar.twig
+//
+// Style guide: navigation.navbar
+.navbar {
+  font-size: 1.05em;
+  z-index: 5;
```

Comment:
Nyt vasta tuli mieleen mut ei se z-index itekseen auta, pitäis olla myös position relative.

Mut käytännössä se tulee kyl siihen päälle kun se on sisällä oleva elementti, ja toi margin -1px siirtää sen samaan kohtaan.

Summa summarum: Ottaisin z-indexin vaan pois jos/kun toimii ilmankin.